packages/frontend/src/components/AudioRecorder/styles.module.scss

```diff
@@ -0,0 +1,132 @@
+.microphoneButton {
+  height: 40px;
+  width: 40px;
+  padding-left: 2px;
+  padding-bottom: 8px;
+  background-color: var(--composerBg);
+  border: none;
+
+  &:focus {
+    outline: none;
+  }
```

Comment:
```suggestion
```

This is bad for accessibility. Why is it needed? We already have `:focus-visible` selector for all buttons

https://github.com/deltachat/deltachat-desktop/blob/e5d32b6a6736d144442c8371339247dcf013877b/packages/frontend/scss/_global.scss#L84-L86